### DISCUSSION TITLE
Local CA (between hosts)

### DISCUSSION THREAD

**Discussion by blaggacao** [nrdxp/nixdev#178] [Ideas]
For if we once require a local CA to establish trust between hosts, this _might_ be a starting point: https://github.com/numtide/devshell/pull/74

**Comment by nrdxp** [comment]
Probably best solution https://www.tweag.io/blog/2020-12-16-trustix-announcement

**Comment by blaggacao** [comment]
Another discovery in this problem space: https://github.com/NixOS/nixpkgs/pull/112322

**Comment by blaggacao** [comment]
I think we can lock this discussion and continue in #181, which greatly overlaps, going forward.
